### PR TITLE
Processing data models

### PR DESCRIPTION
**Pull request by Estherstarr**
In this PR, I implemented the following
- [ ] Create models
- [ ]  use models to insert data
- [ ]  Create custom methods

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :comments
+  has_many :likes
+  has_many :posts
```
Comment (Ismailco) [inline app/models/user.rb]: - [ ][OPTIONAL] Kindly consider adding the foreign_key to the has_many
e.g
```ruby
has_many :posts, foreign_key: 'author_id'
has_many :comments, foreign_key: 'author_id'
has_many :likes, foreign_key: 'author_id'
```

File: app/models/post.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+class Post < ApplicationRecord
+  belongs_to :user
```
Comment (Ismailco) [inline app/models/post.rb]: - [ ] The post belongs to the author, kindly consider implementing this so the relations are setup correctly
e.g ``belongs_to :author, class_name: 'User'``

File: app/models/like.rb [hunk 3]
```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :user
```
Comment (Ismailco) [inline app/models/like.rb]: - [ ] The like belongs to the author, kindly consider implementing this so the relations are setup correctly
e.g ``belongs_to :author, class_name: 'User'``

File: app/models/comment.rb [hunk 4]
```diff
@@ -0,0 +1,11 @@
+class Comment < ApplicationRecord
+  belongs_to :user
```
Comment (Ismailco) [inline app/models/comment.rb]: - [ ] The comment belongs to the author, kindly consider implementing this so the relations are setup correctly
e.g ``belongs_to :author, class_name: 'User'``

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @Estherstarr, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Estherstarr** [comment]
Fix implementations

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @Estherstarr👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**